lib/galaxy/tools/parameters/basic.py

```diff
@@ -333,11 +333,7 @@ def __init__(self, tool, input_source):
 
     def to_json(self, value, app, use_security):
         """Convert a value to a string representation suitable for persisting"""
-        if value is None:
-            rval = "" if not self.optional else None
-        else:
-            rval = unicodify(value)
-        return rval
+        return unicodify(value)
```

Comment:
We could just remove the whole method and rely on the parent's one, which is identical.